community-criteria/open-to-contributions.md

```diff
@@ -49,6 +49,7 @@ order: 4
 ## Developers and designers: what you need to do
 
 * Respond promptly to requests
+* Keep your management appraised of the time and resources you require to support other contributors
```

Comment:
Reading the word 'appraised' and noticing I have no idea what it means, I would suggest describing it in 'plainer english'.